src/conversions/num_bigint.rs

```diff
@@ -82,14 +88,15 @@ unsafe fn extract(ob: &PyLong, buffer: &mut [c_uchar], is_signed: c_int) -> PyRe
     };
     let bytes_obj = ob
         .getattr(intern!(py, "to_bytes"))?
-        .call((buffer.len(), "little"), kwargs)?;
+        .call((length * 4, "little"), kwargs)?;
     let bytes: &PyBytes = bytes_obj.downcast_unchecked();
-    buffer.copy_from_slice(bytes.as_bytes());
-    Ok(())
+    let bytes_u32 = std::slice::from_raw_parts(bytes.as_bytes().as_ptr() as *const u32, length);
```

Comment:
Nits: Could you `use` either `slice` or `from_raw_parts` and use `.cast()` to change the pointer type?

Reply:
sure!